mantidproject/mantid
# Make run1:run2 work in PD reduction GUI

Description of work.

**To test:**

Create an XML file what is below.  Load it with PD reduction GUI to see whether it works.

```
<Reduction>
  <instrument_name>PG3</instrument_name>
  <timestamp>Wed Feb 15 10:17:17 2017</timestamp>
  <python_version>2.7.5 (default, Aug  2 2016, 04:20:16) 
[GCC 4.8.5 20150623 (Red Hat 4.8.5-4)]</python_version>
  <platform>Linux</platform>
  <architecture>('64bit', 'ELF')</architecture>
  <mantid_version>3.9.20170214.1949</mantid_version>
<RunSetup>
  <runnumber>22029,22030,22041,22042,22845:22852</runnumber>
  <sum>0</sum>
  <calibrationfile>/SNS/PG3/shared/CALIBRATION/2015_1_11A_CAL/PG3_PAC_JANIS_d21989_2015_01_25.cal</calibrationfile>
  <characterizationrunsfile>/SNS/PG3/shared/CALIBRATION/2015_1_11A_CAL/PG3_char_2015_01_27-HR.txt</characterizationrunsfile>
  <expinifilename></expinifilename>
  <binning>-0.0008000</binning>
  <resamplex></resamplex>
  <binindspace>1</binindspace>
  <saveas>gsas</saveas>
  <outputdirectory>/SNS/users/wzz/Temp</outputdirectory>
  <finaldataunits>dSpacing</finaldataunits>
  <backgroundnumber></backgroundnumber>
  <vanadiumnumber></vanadiumnumber>
  <vanadiumnoisenumber></vanadiumnoisenumber>
  <vanadiumbackgroundnumber></vanadiumbackgroundnumber>
  <disablebackgroundcorrection>1</disablebackgroundcorrection>
  <disablevanadiumcorrection>0</disablevanadiumcorrection>
  <disablevanadiumbackgroundcorrection>0</disablevanadiumbackgroundcorrection>
  <doresamplex>0</doresamplex>
</RunSetup>
<AdvancedSetup>
 <unwrapref></unwrapref>
 <lowresref></lowresref>
 <cropwavelengthmin></cropwavelengthmin>
 <cropwavelengthmax></cropwavelengthmax>
 <removepromptpulsewidth>50.0</removepromptpulsewidth>
 <maxchunksize></maxchunksize>
 <stripvanadiumpeaks>1</stripvanadiumpeaks>
 <vanadiumfwhm></vanadiumfwhm>
 <vanadiumpeaktol></vanadiumpeaktol>
 <vanadiumsmoothparams></vanadiumsmoothparams>
 <filterbadpulses>10.0</filterbadpulses>
 <backgroundsmoothparams></backgroundsmoothparams>
 <pushdatapositive>AddMinimum</pushdatapositive>
 <preserveevents>1</preserveevents>
 <outputfileprefix></outputfileprefix>
 <scaledata>100.0</scaledata>
</AdvancedSetup>
<FilterSetup>
  <titleofsplitters>Chop by sample temperature with 2K step</titleofsplitters>
  <filterbytimemin>908.686803</filterbytimemin>
  <filterbytimemax>17267.337003</filterbytimemax>
  <filtertype>ByLogValue</filtertype>
  <lengthoftimeinterval></lengthoftimeinterval>
  <numberoftimeinterval>None</numberoftimeinterval>
  <unitoftime>Seconds</unitoftime>
  <logname>SampleTemp340</logname>
  <minimumlogvalue>19.0</minimumlogvalue>
  <maximumlogvalue>301.0</maximumlogvalue>
  <logvalueinterval>2.0</logvalueinterval>
  <filterlogvaluebychangingdirection>Both</filterlogvaluebychangingdirection>
  <timetolerance></timetolerance>
  <logboundary>Centre</logboundary>
  <logvaluetolerance></logvaluetolerance>
  <logvaluetimesections>1</logvaluetimesections>
</FilterSetup>
</Reduction>

```



Fixes #18774.



---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
